defaults:
    timeout: 30
    yaw_threshold: 2
    surge_speed: 0.2
    
